recipes/libcufile/meta.yaml

```
@@ -0,0 +1,160 @@
+{% set name = "libcufile" %}
+{% set version = "1.5.0.59" %}
+{% set cuda_version = "12.0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set target_name = "x86_64-linux" %}  # [linux64]
+{% set target_name = "sbsa-linux" %}  # [aarch64]
+{% set extension = "tar.xz" %}  # [not win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/{{ name }}/{{ platform }}/{{ name }}-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 222c49b29afb544c69542c4e658b0a1bd01fd2306f2136ad6990ccfff482a097  # [linux64]
+  sha256: 4e8d2b525322be9ab6ae574bd1d0325a4bf9c6c3bbdaa1611b5b969b7b34b0c4  # [aarch64]
+  patches:
+    - Makefile.patch
+
+build:
+  number: 0
+  skip: true  # [not linux64]
```

I see `aarch64` selector logic here. If this package supports `aarch64` we need to update this so it isn't skipped:
```suggestion
  skip: true  # [not linux]
```